feature/gnmi/ate_tests/telemetry_interface_packet_counters_test/telemetry_interface_packet_counters_test.go

```diff
@@ -84,6 +84,10 @@ func TestInterfaceCounters(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
 	dp := dut.Port(t, "port1")
 
+	// Await for operational state of the interface
+	t.Logf("Wait for operation state of interface: %s", dp.Name())
+	dut.Telemetry().Interface(dp.Name()).OperStatus().Await(t, 2*time.Minute, telemetry.Interface_OperStatus_UP)
```

Comment:
There is no need to check OperStatus as it is not requited for this test case.

For example, for the following interface with oper-status: DOWN, the counters are still available.
==============

/openconfig-interfaces:interfaces/interface.15/state/admin-status: UP
/openconfig-interfaces:interfaces/interface.15/state/arista-intf-augments:inactive: false
/openconfig-interfaces:interfaces/interface.15/state/counters/in-broadcast-pkts: 0
/openconfig-interfaces:interfaces/interface.15/state/counters/in-discards: 0
/openconfig-interfaces:interfaces/interface.15/state/counters/in-errors: 0
/openconfig-interfaces:interfaces/interface.15/state/counters/in-fcs-errors: 0
/openconfig-interfaces:interfaces/interface.15/state/counters/in-multicast-pkts: 0
/openconfig-interfaces:interfaces/interface.15/state/counters/in-octets: 0
/openconfig-interfaces:interfaces/interface.15/state/counters/in-pkts: 0
/openconfig-interfaces:interfaces/interface.15/state/counters/in-unicast-pkts: 0
/openconfig-interfaces:interfaces/interface.15/state/counters/out-broadcast-pkts: 0
/openconfig-interfaces:interfaces/interface.15/state/counters/out-discards: 0
/openconfig-interfaces:interfaces/interface.15/state/counters/out-errors: 0
/openconfig-interfaces:interfaces/interface.15/state/counters/out-multicast-pkts: 0
/openconfig-interfaces:interfaces/interface.15/state/counters/out-octets: 0
/openconfig-interfaces:interfaces/interface.15/state/counters/out-pkts: 0
/openconfig-interfaces:interfaces/interface.15/state/counters/out-unicast-pkts: 0
/openconfig-interfaces:interfaces/interface.15/state/ifindex: 4037
/openconfig-interfaces:interfaces/interface.15/state/last-change: 1657453340797152000
/openconfig-interfaces:interfaces/interface.15/state/management: false
/openconfig-interfaces:interfaces/interface.15/state/mtu: 0
/openconfig-interfaces:interfaces/interface.15/state/name: Ethernet4/3/1
/openconfig-interfaces:interfaces/interface.15/state/openconfig-platform-port:hardware-port: Ethernet4/3
/openconfig-interfaces:interfaces/interface.15/state/openconfig-platform-transceiver:physical-channel.0: 1
/openconfig-interfaces:interfaces/interface.15/state/openconfig-platform-transceiver:transceiver: Ethernet4/3 transceiver
/openconfig-interfaces:interfaces/interface.15/state/oper-status: DOWN
/openconfig-interfaces:interfaces/interface.15/state/type: iana-if-type:ethernetCsmacd
/openconfig-interfaces:interfaces/interface.15/subinterfaces/subinterface.0/index: 0
/openconfig-interfaces:interfaces/interface.15/subinterfaces/subinterface.0/openconfig-if-ip:ipv4/config/enabled: true
/openconfig-interfaces:interfaces/interface.15/subinterfaces/subinterface.0/openconfig-if-ip:ipv4/state/enabled: true